Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/Page.cs

```diff
@@ -56,5 +56,82 @@ public static UIStatusBarAnimation PreferredStatusBarUpdateAnimation(this IPlatf
 			SetPreferredStatusBarUpdateAnimation(config.Element, value);
 			return config;
 		}
+
+		public static readonly BindableProperty UseSafeAreaProperty = BindableProperty.Create(nameof(UseSafeArea), typeof(bool), typeof(Page), false);
```

Comment:
what does this do ? constrain the page inside the safe area ? if so, it prevents background color or images.

IMHO, the plat specific SafeArea should expose the safe rect to the user. the user can then choose to use that as page padding or deconstruct it to place his objects.

the same SafeArea should probably also default, on iOS, to "0,20,0,0" on root content pages on non-iphoneX devices